src/tests/class_tests/openms/source/DIAPrescoring_test.cpp

```diff
@@ -271,7 +284,103 @@ START_SECTION ( test score function with shifted first transition )
 }
 END_SECTION
 
+START_SECTION ( test score function missing first transition due to different ion mobility )
+{
+  OpenSwath::LightTransition mock_tr1;
+  mock_tr1.product_mz = 500.;
+  mock_tr1.fragment_charge = 1;
+  mock_tr1.transition_name = "group1";
+  mock_tr1.library_intensity = 5.;
+
+  OpenSwath::LightTransition mock_tr2;
+  mock_tr2.product_mz = 600.;
+  mock_tr2.fragment_charge = 1;
+  mock_tr2.transition_name = "group2";
+  mock_tr2.library_intensity = 5.;
+
+
+  double PRECURSOR_ION_MOBILITY = 7;
+  double ION_MOBILITY_WIDTH = 2;
+
+  OpenSwath::SpectrumPtr sptr = (OpenSwath::SpectrumPtr)(new OpenSwath::Spectrum);
+
+  std::vector<OpenSwath::BinaryDataArrayPtr> binaryDataArrayPtrs;
+  OpenSwath::BinaryDataArrayPtr data1(new OpenSwath::BinaryDataArray);
+  OpenSwath::BinaryDataArrayPtr data2(new OpenSwath::BinaryDataArray);
+  OpenSwath::BinaryDataArrayPtr data3(new OpenSwath::BinaryDataArray);
+
+  static const double arr1[] = {
+
+    10, 20, 50, 100, 50, 20, 10, // peak at 499
+    3, 7, 15, 30, 15, 7, 3,      // peak at 500
+    1, 3, 9, 15, 9, 3, 1,        // peak at 501
+    3, 9, 3,                     // peak at 502
+
+
+    10, 20, 50, 100, 50, 20, 10, // peak at 600
+    3, 7, 15, 30, 15, 7, 3,      // peak at 601
+    1, 3, 9, 15, 9, 3, 1,        // peak at 602
+    3, 9, 3                      // peak at 603
+  };
+  std::vector<double> intensity (arr1, arr1 + sizeof(arr1) / sizeof(double) );
```

Comment:
no need for these tricks since C++11
Just do: `std::vector<double> intensity { 1, 2, 3, 4, 5 ...};`